docs/changelog.md

```diff
@@ -1,9 +1,10 @@
-0.15.1.4
+dev
 
 - Improved error reporting during venv metadata inspection.
 - [bugfix] Fixed incompatibility with pypy as venv interpreter (#372).
 - [bugfix] Replaced implicit dependency on setuptools with an explicit dependency on packaging (#339).
 - [refactor] Moved all commands to separate files within the commands module (#255).
+- [bugfix] Continue reinstalling packages after failure
```

Comment:
As of now, there is no enforced convention here. My preference is to not use periods on sentence fragments, but it seems like a discussion on this topic is destined for bikeshedding. 

I'm going to merge as-is since the changelog history is already inconsistent. If someone wants to make the entire changelog consistent and add a linter that breaks the build, that will be fine. Otherwise I'm just going to let contributors continue to write it in freeform as we have been.